src/odemis/driver/test/orsay_test.py

```diff
@@ -243,7 +248,7 @@ def test_errorstate(self):
         """
         Check that the state VA is updated properly
         """
-        if not TEST_NOHW == "sim":
+        if TEST_NOHW == 1:
```

Comment:
that should be `is True`. It works because in Python True == 1, but it looks odd.